rasa/cli/test.py

```diff
@@ -53,7 +54,7 @@ def add_subparser(
 
     test_core_parser.set_defaults(func=test_core)
     test_nlu_parser.set_defaults(func=test_nlu)
-    test_parser.set_defaults(func=test)
+    test_parser.set_defaults(func=test, stories=DEFAULT_E2E_TESTS_PATH)
```

Comment:
But this way `rasa --help` would still show the `--e2e` flag although you can't use it, right? We should remove this flag from `test_parser` and only keep it for `test_core_parser`

Reply:
True. Fixing it